app/views/images/index.html.erb

```diff
@@ -1,13 +1,13 @@
-<%= stylesheet_link_tag "image" %>
-
-<h1>Welcome!</h1>
-
-<%= link_to 'New image', new_image_path %>
-
-<table class="image_table">
-  <% @images.each do |image| %>
-      <tr>
-        <td><%= image_tag image.imageurl, class: "index_image" %></td>
-      </tr>
-  <% end %>
-</table>
+<div class="album">
```

Comment:
In order to follow BEM format, I would call the outer parent div class ```album--container```
and the inner card ```album--card``` and the image ```album--card-img``` notice how I use the -- to separate the parent prefix from the child